jmeter-java-dsl-graphql/src/test/java/us/abstracta/jmeter/javadsl/graphql/DslGraphqlSamplerTest.java

```diff
@@ -137,6 +138,41 @@ public void shouldSendGraphqlQueryToServerWhenGraphqlSamplerWithHttpGet()
     ).run();
     verify(getRequestedFor(anyUrl()).withQueryParam("query", equalTo(QUERY)));
   }
+  @Test
+  public void shouldThrowExceptionForInvalidVariable() {
+    DslGraphqlSampler sampler = graphqlSampler("http://localhost", "{ user(id: 1){ name } }");
+    assertThrows(IllegalArgumentException.class, () -> sampler.variable("invalidVar", new Object()));
+  }
+
+  @Test
+  public void shouldHandleLongOperationName() throws Exception {
+    String longOperationName = "a".repeat(300);
+    testPlan(
+            threadGroup(1, 1,
+                    graphqlSampler(wiremockUri, QUERY)
+                            .operationName(longOperationName)
+            )
+    ).run();
+    verify(postRequestedFor(anyUrl())
+            .withRequestBody(equalToJson(buildRequestBody(longOperationName, null))));
+  }
+  @Test
+  public void shouldSendGraphqlQueryToServerWhenGraphqlSamplerWithHttpPut() throws Exception {
+    testPlan(
+            threadGroup(1, 1,
+                    graphqlSampler(wiremockUri, QUERY)
+                            .httpGet()
+            )
+    ).run();
+    verify(getRequestedFor(anyUrl()).withQueryParam("query", equalTo(QUERY)));
+  }
+  @Test
```

Comment:
I am not sure this is an actual functionality to test for. I mean, the sampler will probably fail when the request is sent to the server in such scenario. We are just not adding validation here in the code, which makes it more flexible but also more likely to fail afterwards. We could add such logic, but instead of that I would just leave it untested, and if a user ever complains about this behavior, then add validation and change test.